tests/foreman/ui/test_adusergroups.py

```diff
@@ -0,0 +1,134 @@
+"""Test class for Active Directory Feature"""
+from robottelo.test import UITestCase
+
+
+class ADUserGroups(UITestCase):
+    """Implements Active Directory feature tests in UI."""
+
+    def test_create_ldap_auth_withAD(self):
+        """@Test: Create LDAP authentication with AD
+
+        @Feature: Active Directory - create LDAP AD
+
+        @steps:
+
+        1. Create a new LDAP Auth source with AD
+
+        2. Fill in all the fields appropriately for AD
```

Comment:
Can you drop the space between each list item?

    1. Create a new LDAP Auth source with AD
    2. Fill in all the fields appropriately for AD